src/Relations/BelongsToThrough.php

```diff
@@ -91,7 +91,7 @@ public function __construct(
     /**
      * Set the base constraints on the relation query.
      *
-     * @return void
+     * @inheritDoc
      */
```

Comment:
This should be the entire doc:

```
/** @inheritDoc */

Reply:
Applied on a couple places where it works without introducing phpstan errors. The remaining usages that are multiline causes phpstan errors when updating to `/** @inheritDoc */`

Reply:
What are the errors?

You can't use `@inheritDoc` in the middle of an existing doc (there is an inline `{@inheritDoc}` variant but we shouldn't be using that`

Reply:
See https://github.com/staudenmeir/belongs-to-through/pull/102#discussion_r1743909331